### ISSUE TITLE
Cli: Issue with delimiter on Windows

### ISSUE THREAD

**Issue by Marr11317** [messageformat/messageformat#343]
Given the following dir structure:
```
-package.json
-messages/
    -en.json
    -fr.json
    -es.json
```
compiling the translation files with 

```
messageformat messages/ --locale=en,fr,es --outfile=messages.js
```

creates object with the full path to the locale files.

```js
...
export default {
  "F:/prog/messageformat/messages/en": { // HERE'S THE PROBLEM
    // translation keys
    ...
  },
  "F:/prog/messageformat/messages/es": { // AND HERE TOO
    ...
  },
  "F:/prog/messageformat/messages/fr": { // DITO
    ...
  }
}
```

wanted: 
```js
...
export default {
  "en": {
    // translation keys
    ...
  },
  "es": { 
    ...
  },
  "fr": { 
    ...
  }
}
```

I tried every thing I could think of for the delimiter, with no luck. 

Is there something I am missing?

**Comment by Marr11317** [comment]
[Sample repo](https://github.com/marr11317/messageformat-issue343)

**Comment by Marr11317** [comment]
Note that I juste tested and it works perfectly under Ubuntu 20.04.3 LTS

**Comment by eemeli** [comment]
It looks like something wasn't really working with the platform-specific handling of `--delimiters`, so I removed it. Now both `/` and `\` are always considered as delimiters by default.

We really ought to have tests for the CLI, esp. as GitHub Actions allows for running them in various OS environments. That's actually how I duplicated this myself and verified that the fix works: https://github.com/messageformat/messageformat/runs/3636288526
